clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/MigrateLoadBalancerStrategy.java

```diff
@@ -216,14 +216,23 @@ private void applyListenerPolicies(AmazonElasticLoadBalancing sourceClient, Amaz
         .withLoadBalancerName(loadBalancerName)
     ).getPolicyDescriptions();
     sourcePolicies.removeIf(p -> targetPolicies.stream().anyMatch(tp -> tp.getPolicyName().equals(p.getPolicyName())));
-    sourcePolicies.forEach(p -> targetClient.createLoadBalancerPolicy(
-      new CreateLoadBalancerPolicyRequest()
+    sourcePolicies.forEach(p -> {
+      CreateLoadBalancerPolicyRequest request = new CreateLoadBalancerPolicyRequest()
         .withPolicyName(p.getPolicyName())
-      .withLoadBalancerName(loadBalancerName)
-      .withPolicyTypeName(p.getPolicyTypeName())
-      .withPolicyAttributes(p.getPolicyAttributeDescriptions().stream().map(d ->
-        new PolicyAttribute(d.getAttributeName(), d.getAttributeValue())).collect(Collectors.toList()))
-    ));
+        .withLoadBalancerName(loadBalancerName)
+        .withPolicyTypeName(p.getPolicyTypeName());
+      // Only include ciphers if this is not a pre-defined policy, which is determined by the presence of this attribute
```

Comment:
"ciphers" is a bit misleading (I believe the policy attributes are ciphers and allowed protocols?).

Might be clearer as `// only copy policy attributes if this is not a pre-defined policy (as defined by the presence of 'Reference-Security-Policy'`